python/lsst/ts/standardscripts/maintel/laser_tracker/align.py

```diff
@@ -0,0 +1,206 @@
+# This file is part of ts_standardscripts
+#
+# Developed for the LSST Telescope and Site Systems.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+
+__all__ = ["Align"]
+
+import yaml
+
+from lsst.ts import salobj
+from lsst.ts.idl.enums.LaserTracker import LaserStatus, LaserTrackerComponent
+from lsst.ts.observatory.control import RemoteGroup
+from lsst.ts.observatory.control.maintel.mtcs import MTCS, MTCSUsages
+from lsst.ts.observatory.control.remote_group import Usages
+
+
+class Align(salobj.BaseScript):
+    """Set up Laser Tracker.
+
+    Parameters
+    ----------
+    index : `int`
+        Index of Script SAL component.
+
+    Notes
+    -----
+    **Checkpoints**
+    - "Starting alignment procedure.": Starting alignment with laser tracker.
+    - "M2 Hexapod aligned with laser tracker.": M2 aligned.
+    - "Camera Hexapod aligned with laser tracker.": M2 aligned.
+    """
+
+    def __init__(self, index: int, add_remotes: bool = True):
+        super().__init__(index, descr="Align MTCS components with laser tracker.")
+
+        self.laser_tracker = RemoteGroup(
+            domain=self.domain,
+            components=["LaserTracker:1"],
+            intended_usage=None if add_remotes else Usages.DryTest,
+            log=self.log,
+        )
+
+        self.mtcs = MTCS(
+            domain=self.domain,
+            intended_usage=None if add_remotes else MTCSUsages.DryTest,
+            log=self.log,
+        )
+
+        self.timeout_align = 120
+        self.timeout_std = 60
+
+        self.max_iter = 10
+        self.tolerance_linear = 1.0e-7 #meter
+        self.tolerance_angular = 5.0 / 3600.0 #degrees
+        self.target = 'M2'
```

Comment:
This should either be `None` or `LaserTrackerComponent.M2`

Reply:
Done